### DISCUSSION TITLE
Add singbox core as a choice `in Settings

### DISCUSSION THREAD

**Discussion by ebkarimi** [2dust/v2rayNG#2569] [Ideas]
v2rayNG is the best v2ray client for Android but it only uses Xray core. Please add sing box as a choice in the Settings. Sing box core is better in many situations than Xray.

**Comment by fransewoo** [comment]
+1，同求大佬更新可以支持sing_box内核切换。

**Comment by soraclub** [comment]
+1, 同求支持sing-box

**Comment by ghost** [comment]
同求。
